BootloaderCommonPkg/Library/BootloaderDebugLib/DebugLib.c

```diff
@@ -73,14 +74,17 @@ DebugPrint (
   VA_END (Marker);
 
   //
-  // Send the print string to a Serial Port
+  // Send the print string to debug output handler
   //
   if (PcdGet32 (PcdDebugOutputDeviceMask) & DEBUG_OUTPUT_DEVICE_LOG_BUFFER) {
     DebugLogBufferWrite  ((UINT8 *)Buffer, AsciiStrLen (Buffer));
   }
   if (PcdGet32 (PcdDebugOutputDeviceMask) & DEBUG_OUTPUT_DEVICE_SERIAL_PORT) {
     SerialPortWrite ((UINT8 *)Buffer, AsciiStrLen (Buffer));
   }
+  if (PcdGet32 (PcdDebugOutputDeviceMask) & DEBUG_OUTPUT_DEVICE_CONSOLE) {
+    ConsoleWrite ((UINT8 *)Buffer, AsciiStrLen (Buffer));
```

Comment:
if serial is also in console out device,  will it cause issue in output ?

Reply:
If serial and console are enabled, and console out has serial enabled, all messages sent via DebugLib will be delivered on the serial port twice.